controllers/reactController.ts

```diff
@@ -6,7 +6,7 @@ const getIndex = (req: Request, res: Response): void => {
   res.header('Cache-Control', 'no-cache, no-store, must-revalidate');
   res.header('Pragma', 'no-cache');
   res.header('Expires', '0');
-  res.sendFile(path.join(__dirname, '..', 'frontend', 'dist', 'index.html'));
+  res.sendFile(path.join(__dirname, '..', 'frontend', 'index.html'));
```

Comment:
This change seems ok to me, because postbuild does `cp -a frontend/dist dist/frontend` but how was this working before?